engineering/README.md

```diff
@@ -0,0 +1,155 @@
+# Blockstack Engineering Handbook
+
+Welcome to the Blockstack Engineering Handbook! This is our attempt at building an engineering process as we build a new internet for decentralized apps.
+
+Here you'll find notes, checklists and knowledge that w
+
+## Table of Contents
+
+- [Sprints](#sprints)
+- [Pull Requests](#pull-requests)
+- [Repository Management](#repository-management)
+- [Planning](#planning)
+- [Meetings](#meetings)
+
+## Sprints 
+We operate with 2 week sprints. At the beginning of each two week sprint, we have a weekly meeting followed by breakout sessions for the product and engineering teams where user stories and sprint items are generated.
+
+Mid-sprint, we have a weekly meeting and prune tasks from the sprint.
+
+### Sprint management
+We create a new Github organization kanban for each sprint with 5 columns: 
+* User Stories
+* To Do
+* Doing
+* Delivered
+* Accepted.
+
+#### User Stories
+
+Create an issue for each user story in this column. The title  of the story should generally be in the format 'As a `<stakeholder>` I can `<do something>`.
+
+Each user story is assigned to a team member who is the owner of that user story for the sprint. The owner of the user story is responsible for breaking the user story into discrete tasks. Each of these tasks should be linked from the user story. These tasks are added to the To Do column of the sprint board.
+
+#### To Do
+
+**Responsible party:** User story assignee
+
+Tasks that need to be completed to satisfy the user stories belong here until someone starts working on them.
+
+This column uses the "To Do" Github automation enabling the options to put newly opened issues and reopened into this column automatically.
+
+#### Doing
+**Responsible party:** Issue assignee
+ 
+Move issues from the To Do to column into this column when you're working on them.
+
+#### Delivered 
+**Responsible party:** Issue assignee
+
+When you believe you've completed an issue, mark it as closed and move it to this column. You should also send a pull request for the issue to `develop` or `master` as appropriate and place that pull request in this column.
+
+#### Accepted
+**Responsible party:** Pull request reviewer
+
+The reviewer of a pull request is moves the pull request and any issues that it satisfies to the Accepted column when the pull request is approved.
+
+Issues that don't pass review should be moved by the pull request reviewer back to the To Do column.
+
+
+## Pull Requests
+### Creating a pull request
+**Responsible party:** Issue assignee
+
+Pull requests are made from topic (feature, hotfix, etc) branches to `develop` or `master`. We never commit directly to these two branches.
+
+#### Checklist
+[ ] A description of what is included in the pull request
+[ ] Links to any other pull requests in other repos that are required for this pull request
+[ ] Instructions on how to test this pull request
```

Comment:
This checklist (and the others without leading spaces before the boxes) doesn't render correctly for me

Reply:
Thanks for catching this